### DISCUSSION TITLE
Changing volume in Small Increments?

### DISCUSSION THREAD

**Discussion by zsteinkamp** [MonitorControl/MonitorControl#547] [Q&A] [answered]
Great software, thanks for putting it together. It's kind of mind-boggling that Apple doesn't support DDC natively. Anyhow ... 

My monitor has a 0-100 volume adjustment range. For my listening purposes, 0-15 are really the usable range of values. Using the volume up/down keys on the keyboard changes the volume in increments of 6. I was hoping for a more fine control over the range, and so I looked at the source code. I see mention of `isSmallIncrement` in the code, presumably with Shift or Option triggering that value. 

https://github.com/MonitorControl/MonitorControl/blob/d46c429c075c6932ace7aa3eb25b669332f86665/MonitorControl/AppDelegate.swift#L341

However, holding shift while pressing the volume keys does not change the step size. 

Is there something I'm missing there? If not, and there is interest, I could look at adding a range or step preference.

Thanks!

**Comment by zsteinkamp** [comment, marked as answer]
My bad ... Option+Shift does work. I misunderstood the code. Thanks!

**Comment by waydabber** [comment]
Hi @zsteinkamp - we also plan to have the ability to limit the volume range - see this thread: https://github.com/MonitorControl/MonitorControl/issues/310

**Comment by locksoft** [comment]
Sorry to open a new thread, I've missed this one.
That looks really great. So, for the moment, I'll try to find a way to tweak the volume buttons to trigger the fine tuning (but a native option for smaller steps and limit would be really top). :)

**Comment by zsteinkamp** [comment]
The "Use fine OSD scale for volume" option in 4.0.0b1 is perfect! Thanks!
<img width="421" alt="Screen Shot 2021-09-30 at 3 26 21 PM" src="https://user-images.githubusercontent.com/8418898/135538286-f0e71057-5426-4395-bd0d-21e74aa151c0.png">
 "

**Reply by waydabber** [reply]
You are welcome! :)
